.circleci/config.yml

```diff
@@ -182,6 +182,24 @@ jobs:
           at: ~/
       - run: ../../.circleci/test-js-client.sh
 
+  docs:
+    docker:
+      - image: circleci/node:10
+    working_directory: ~/project
+    steps:
+      - attach_workspace:
+          at: ~/
+      - add_ssh_keys:
+          fingerprints:
+            - "08:fc:2b:fb:06:0e:8f:0f:01:1f:28:86:83:89:11:28"
```

Comment:
Fwiw, this is the fingerprint of a deploy key that I physically deleted after adding to Circle/GitHub, so only CircleCI will be able to use it (because obviously, we don't want stray deploy keys leaking out anywhere).